source/sass/61-layouts/_detail-layout.scss

```diff
@@ -18,4 +18,16 @@
       }
     }
   }
+
+  // @TODO: remove this when the styleguide version is upgraded.
```

Comment:
Todo?

Reply:
They're theming overrides for bugs that are still present in the styleguide. I could extract them into a separate file as well, if you find that more clear? 🤔